src/index.js

```diff
@@ -138,6 +140,8 @@ function createLoadableComponent(loadFn, options) {
     constructor(props) {
       super(props);
       init();
+      CONSTRUCTED_INITIALIZERS.push(init);
+      CONSTRUCTED_RESULTS.push(res);
```

Comment:
What is the difference (in terms of program execution) between pushing these items at class/constructor level instead of doing it at module/file level (e.g. `ALL_INITIALIZERS` and `READY_INITIALIZERS`?

Reply:
The difference is that at class construction we know they are actually being used in a render.